GitUtils.ps1

```diff
@@ -115,12 +115,15 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
         $filesModified = @()
         $filesDeleted = @()
         $filesUnmerged = @()
+        $stashCount = 0
 
         if($settings.EnableFileStatus -and !$(InDisabledRepository)) {
             dbg 'Getting status' $sw
             $status = git -c color.status=false status --short --branch 2>$null
+            $stashCount = $null | git stash list 2>$null | measure-object | select -expand Count

```

Comment:
> Pipe null into git stash command because otherwise it grabs everything in stdin - annoying if you're in the habit of typing before you get the next command prompt.

Piping `$null` into `git stash` is...curious. It seems like the problem might actually be the pager - try `git --no-pager list`?